pkg/apb/types.go

```diff
@@ -50,6 +50,16 @@ type ExtractedCredentials struct {
 	// might be more one day
 }
 
+// HACK: created these State types because I couldn't import broker into the
```

Comment:
I'm going to remove the HACK keyword. After using these states, I'm ok with them.